build/tasks/compare_size.mjs

```diff
@@ -142,7 +152,9 @@ export async function compareSize( { cache = ".sizecache.json", files } = {} ) {
 	console.log( output );
 
 	// Always save the last run
+	// Save version under last run
 	sizeCache[ " last run" ] = {
+		meta: { version: VERSION },
```

Comment:
[Suggestion] It's a bit weird that the version of the whole schema is under the ` last run` key instead of in a global `meta` section. You may consider moving that.

Reply:
I considered it, but last run always gets saved and this is just a little smaller, saving another object. Practically, I think the difference is minimal.

Reply:
OK, this was just a suggestion, I don't insist. 🙂